src/test_driver/openiotsdk/unit-tests/main/main.cpp

```diff
@@ -25,6 +26,9 @@
 #include "NlTestLogger.h"
 
 constexpr nl_test_output_logger_t NlTestLogger::nl_test_logger;
+extern "C" {
+#include <mbedtls_threading_cmsis_rtos.h>
```

Comment:
I think we should include threading_alt.h so that if we change the underlying threading implementation includes don't need to be modified

Reply:
main uses this header, even you change threading_alt it wouldn't change that, it's all academic of course since SDK is tied to cmsis-rtos

Reply:
I recomend to use:
```suggestion
#include <mbedtls/threading.h>
```
No C++ guard is required then. And this header includes the `threading_alt.h`